cpu/ppc64_cpu_test.py

```diff
@@ -172,20 +168,21 @@ def dscr(self):
             "/sys/devices/system/cpu/dscr_default").strip(), 16)
         self.equality_check("DSCR", op1, op2)
 
-    @staticmethod
-    def smt_loop():
+    def test_smt_loop(self):
         """
         Tests smt on/off in a loop
         """
         for _ in range(1, 100):
-            process.run("ppc64_cpu --smt=off && ppc64_cpu --smt=on",
-                        shell=True)
+            if process.system("ppc64_cpu --smt=off && ppc64_cpu --smt=on",
```

Comment:
`ppc64_cpu --smt=` is used in many places for better maintainability of code please use a variable.

Reply:
@nasastry Thanks. Done, please have a check now.